pkgs/apache/maven-mvnd/registry.yaml

```diff
@@ -0,0 +1,41 @@
+packages:
+  - type: github_release
+    repo_owner: apache
+    repo_name: maven-mvnd
+    description: Apache Maven Daemon
+    asset: maven-mvnd-{{.Version}}-m39-{{.OS}}-{{.Arch}}.{{.Format}}
+    format: tar.gz
+    files:
+      - name: mvnd
+        src: maven-mvnd-{{.Version}}-m39-{{.OS}}-{{.Arch}}/bin/mvnd
+    version_constraint: semver(">= 1.0.0-m6")
```

Comment:
It seems you didn't follow the guide.

https://aquaproj.github.io/docs/products/aqua-registry/contributing#how-to-add-a-package

Reply:
You are right. I tried to follow it but it didn't work for me. Probably it's because I use Fedora and only Podman (not Docker) is available. Here is what I got when running `cmdx s <package>`:
```
INFO[0000] create a symbolic link                        aqua_version=2.23.1 command=aqua-proxy env=linux/amd64 package_name=aqua-proxy package_version=v1.2.5 program=aqua registry=
+ opts=
+ '[' -n '' ']'
+ '[' -n '' ']'
+ docker exec -ti -w /aqua-registry aqua-registry aqua-registry scaffold apache/maven-mvnd
Error: crun: executable file `aqua-registry` not found in $PATH: No such file or directory: OCI runtime attempted to invoke a command that was not found
exit status 127
```

Reply:
Oh, I see. Thank you for your explanation.
Maybe this issue is related, though I couldn't reproduce the issue.

- https://github.com/aquaproj/aqua-registry/issues/20289